manifests/ccm.yaml

```diff
@@ -78,3 +78,6 @@ subjects:
 - kind: User
   name: cloud-controller-manager
   namespace: kube-system
+- kind: ServiceAccount
```

Comment:
Why do we add service account cloud-controller-manager to the rbac?

Reply:
Changing the cloud-provider args made the built-in cloud-controller-manager want to watch Nodes, which previously lacked permission to.